src/core/components/layout/components/container/styles.ts

```diff
@@ -25,14 +25,18 @@ export const container = css`
 	}
 `;
 
-export const containerBorder = css`
-	border-style: solid;
-	border-color: ${border.secondary};
-	border-width: 0;
+export const containerSideBorders = (color?: string) => css`
+	border-left-style: solid;
+	border-right-style: solid;
+	border-left-color: ${color || border.secondary};
+	border-right-color: ${color || border.secondary};
```

Comment:
I'd like to avoid, or reduce the scope of, dynamic styles as much as possible. Would  you be able to extract the styles that rely on `color` into a separate function and call it only when a `borderColor` is specified in the props?